experimental/CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ All notable changes to experimental packages in this project will be documented
 * feat(exporter-prometheus)!: stop the using `type` field to enforce naming conventions [#5291](https://github.com/open-telemetry/opentelemetry-js/pull/5291) @chancancode
   * Any non-monotonic sums will now be treated as counters and will therefore include the `_total` suffix.
 * feat(shim-opencenus)!: stop mapping removed Instrument `type` to OpenTelemetry metrics [#5291](https://github.com/open-telemetry/opentelemetry-js/pull/5291) @chancancode
-  * The internal OpenTelemetry data model dropped the concept of instrument type on exported metrics, therefore mapping it is not necessary anymore.  
+  * The internal OpenTelemetry data model dropped the concept of instrument type on exported metrics, therefore mapping it is not necessary anymore.
```

Comment:
Reviewer note: This change was unintentional (save in my editor removes trailing newlines). I can revert this if anyone would like.